letsencrypt-auto-source/letsencrypt-auto

```diff
@@ -420,6 +419,20 @@ BootstrapRpmCommon() {
       python-tools
       python-pip
     "
+  # Fedora 26 starts to use the prefix python2 for python2 based packages.
+  # this elseif is theoretically for any Fedora over version 26:
+  elif $SUDO $tool list python2 >/dev/null 2>&1; then
+    pkgs="$pkgs
+      python2
+      python2-libs
+      python2-setuptools
+      python2-devel
+      python2-virtualenv
+      python2-tools
+      python2-pip
+    "
+  # Some distros and older versions of current distros use a "python27"
+  # instead of "python" naming convention. Try both conventions.
```

Comment:
Since we now have 3 conventions, this old "Try both conventions." phrase doesn't make sense. You could change it if you wanted to say something like "Try all conventions" or "Try all three conventions", however, I personally have a preference for just removing it entirely.